java/checkstyle.xml

```diff
@@ -46,11 +47,6 @@
     <property name="fileNamePattern"
               value="io.fury.*MurmurHash3|Identity2IdMap.java"/>
   </module>
-  <module name="LineLength">
```

Comment:
Why remove this LineLength check? a line too long is not friendly for readbility

Reply:
This property is (or should be) covered by spotless, while spotless can omit checks for license header but checkstyle warning on license header (for kyro long line length). The spotless style should be enough.